### PR TITLE
fix(twitch): profile icon not showing

### PR DESCRIPTION
**Pull request by Kathund**
## Description 

Fix issue #8674 were the profile icon dosent show up.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image
image

</details>

### CODE REVIEW

**Comment by Kathund** [comment]
closes #8674

File: websites/T/Twitch/presence.ts [hunk 1]
```diff
@@ -467,7 +467,7 @@ presence.on("UpdateData", async () => {
 						profilePic =
 							document
 								.querySelector<HTMLImageElement>(
-									".tw-halo > .tw-aspect > .tw-avatar > .tw-image-avatar"
+									"#live-channel-stream-information > div > div > div.Layout-sc-1xcs6mc-0.dRGOOY > div > div.Layout-sc-1xcs6mc-0.iXRytg > div > div > div.Layout-sc-1xcs6mc-0.kpHsJz.avatar--t0iT1 > a > div > div.ScAvatar-sc-144b42z-0.dFBjte.tw-avatar > img"
```
Comment (theusaf) [inline websites/T/Twitch/presence.ts:470]: Is it possible to make this query less specific and with no computer-generated class names?

Eg:
```css
#live-channel-stream-information [class*=Layout][class*=avatar] .tw-avatar > img
```

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/T/Twitch/presence.ts (hunk 1 above)
Comment (Kathund) [reply websites/T/Twitch/presence.ts:470]: i just used the raw class names from twitch lmao. I can do some testing when i get a chance to see if i can make it a better query

Comment (theusaf) [reply websites/T/Twitch/presence.ts:470]: Any updates?

**Comment by theusaf** [comment]
Closing for inactivity. You may open a new PR after resolving the issues mentioned in this one.

**Comment by Kathund** [comment]
@theusaf hey sorry I got distracted. Is it possible to just reopen this pr?

**Comment by Kathund** [comment]
@theusaf Don't worry about it i cant even auth my premid to test. https://discord.com/channels/493130730549805057/607524579874832446/1281591465700753491

**Pull request closed without merging**